src/Platform/src/Platform.Windows.cpp

```diff
@@ -120,7 +120,7 @@ namespace OpenLoco::Platform
 
     std::vector<fs::path> getDrives()
     {
-        char drive[4] = { 'A', ':', '\0' };
+        char drive[4] = { 'A', ':', '\\', '\0' };
```

Comment:
Not having the trailing backslash here would lead the `fs::is_directory` call to fail. I imagine vanilla didn't show the trailing slash, but I think it's fine like this as well. If preferred, we can remove it below before invoking `drives.push_back`, though.